### ISSUE TITLE
Fails if path firebase_cli_path contains spaces

### ISSUE THREAD

**Issue by ultimate-deej** [fastlane-community/fastlane-plugin-firebase_app_distribution#11]
It's impossible to run `firebase_app_distribution` action if the value passed to `firebase_cli_path` contains spaces. Failure happens on `Actions.sh_control_output`.

I tried to shellescape the string before passing to action, but in this case the `verify_block` fails.

It only begins to work if I shellescape the path by changing the following line
```ruby
cmd = [params[:firebase_cli_path].chomp, FIREBASECMD_ACTION]
```
to
```ruby
cmd = [params[:firebase_cli_path].chomp.shellescape, FIREBASECMD_ACTION]
```

**Comment by rebehe** [comment]
Hi Maxim, thanks for the feedback! We'll put in a fix and let you know when it's live.

**Comment by rebehe** [comment]
Hi Maxim, this just got fixed in version 0.1.4! Run `fastlane update_plugins` to pull down this latest version
